region/wreckedship/main.json

```diff
@@ -1638,6 +1638,13 @@
                     {"spikeHits": 3}
                   ]
                 },
+                {
+                  "name": "Bowling Spike Run",
+                  "notable": false,
+                  "requires": [
+                    {"spikeHits": 7}
```

Comment:
Looks inconsistent to the other strats.  Should be using no run speed jumps right?

Reply:
3 damage boosts would fit in a strat too.

Reply:
I was doing it as running to the right without jumping. With poor jumps and no run I get 6 hits